galore/cross_sections.py

```diff
@@ -281,3 +303,242 @@ def _eval_fit(energy, coeffs):
+    highest_obital_cross_section = np.max(np.nan_to_num(unit_cross_sections))
+    orbitals_cross_sections_dict['s'] = highest_obital_cross_section
+    
+    
+    
+    ## result for 'p', 'd', 'f' orbitals
+    orbitals = ['p', 'd', 'f']
+    
+    for orbital in orbitals:
+        interm_matrix =  np.array([value for key, value in cross_sections_by_subshells.items() if orbital in key]).T
+        electrons_number  = np.array([value for key, value in electron_counts_by_subshells.items() if orbital in key])
+ 
+        if np.shape(interm_matrix) != (0,):
+            if dataset == 'Scofield':
+                subshells_cross_sections = interm_matrix[energy_index]
+                result = _get_avg_orbital_cross_sections(subshells_cross_sections,electrons_number)
+         
+                ## get highest cross section of this obital
+                highest_obital_cross_section = np.max(np.nan_to_num(result))
+                orbitals_cross_sections_dict[orbital] = highest_obital_cross_section
+                
+            elif dataset == 'Yeh':
+                obital_cross_sections = interm_matrix[energy_index]
+                ## get unit cross sections of this orbital
+                unit_cross_sections = obital_cross_sections/electrons_number
+                ## get highest cross section of this obital
+                highest_obital_cross_section = np.max(np.nan_to_num(unit_cross_sections))
+                orbitals_cross_sections_dict[orbital] = highest_obital_cross_section
+            
+                
+    return orbitals_cross_sections_dict
+
+def _get_metadata(energy,dataset):
+    """
+    Args:
+        energy(float): energy value  
+        dataset(str): 'Scofield' or 'Yeh'
+        
+        Note: 1.'Scofield' for J. H. Scofield (1973)
+                Lawrence Livermore National Laboratory Report No. UCRL-51326              
+              2.'Yeh' for Yeh, J.J. and Lindau, I. (1985) 
+                Atomic Data and Nuclear Data Tables 32 pp 1-155   
+                
+    Returns:
+        metadata_dict: containing the input energy value 
+                       and description of input reference
+  
+    """   
+    
+    metadata_dict = {}
+    metadata_dict['energy'] = energy
+    if dataset == 'Scofield':
+        metadata_dict['citation'] = 'J.H. Scofield, Theoretical photoionization cross sections from 1 to 1500 keV'
+        metadata_dict['link'] = 'https://doi.org/10.2172/4545040' 
+    elif dataset == 'Yeh':
+        metadata_dict['citation'] = 'Yeh, J.J. and Lindau, I. (1985) Atomic Data and Nuclear Data Tables 32 pp 1-155'
+        metadata_dict['link'] = 'https://doi.org/10.1016/0092-640X(85)90016-6'
+    else:
+        metadata_dict('dataset error: you can enter reference as "Scofield" or "Yeh" ' )
+    return metadata_dict
+
+
+
+def galore_install_data(DOI,data_file_dir,data_file_path):
+    
+    
+    if os.path.isfile(data_file_path)== True:
+        print("Data file exists.")
```

Comment:
What if somebody wants to update to a corrected newer version?

If we don't provide some kind of `force=True` option, it would be helpful to print _where_ the file exists so it can be examined/deleted.